packages/components/psammead-media-indicator/src/index.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from 'styled-components';
+import { string } from 'prop-types';
+import { C_WHITE, C_EBON } from '@bbc/psammead-styles/colours';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+import { GEL_SPACING, GEL_SPACING_HLF } from '@bbc/gel-foundations/spacings';
+import { GEL_MINION, GEL_FF_REITH_SANS } from '@bbc/gel-foundations/typography';
+
+const MediaIndicatorWrapper = styled.div`
+  padding: ${GEL_SPACING} ${GEL_SPACING_HLF};
+  background-color: ${C_WHITE};
+  display: inline-block;
+  font-family: ${GEL_FF_REITH_SANS};
+  ${GEL_MINION};
+  color: ${C_EBON};
+`;
+
+const PlayIcon = styled.svg`
+  vertical-align: middle;
+  margin: 0 ${GEL_SPACING_HLF};
+  fill: ${C_EBON};
+`;
+
+const TimeDuration = styled.time`
+  vertical-align: middle;
+  margin: 0 ${GEL_SPACING_HLF};
+`;
+
+const MediaIndicator = ({ datetime, duration, offscreenText }) => (
+  <MediaIndicatorWrapper>
+    <PlayIcon
```

Comment:
We could use flex to align the icon and time vertically. A solution would be wrapping the `PlayIcon` and `TimeDuration` in another `div` like `<FlexWrapper>`, using the following styles:

`const FlexWrapper = styled.div`
  `display: flex;`
  `align-items: center;`
`;`

And removing `vertical-align: middle;` from the `PlayIcon` and `TimeDuration` definitions.

Reply:
This is a nice idea, we may need a fallback for `flex` on say [IE9](https://caniuse.com/#search=flex). So if we followed a progressive enhancement approach we should have:
```
const FlexWrapper = styled.div`
  vertical-align: middle;

  @supports (display: flex) {
    display: flex;
    align-items: center;
    vertical-align: baseline; // default value
  }
`;
```

Reply:
I'm assuming in the above that `vertical-align: middle;` is inherited by the children of `FlexWrapper` (I haven't checked). But if it isn't we could target the children both outside and inside of the `@supports` check

Reply:
Yeah I like that

Reply:
Nice one :ok_hand: